cfme/fixtures/service_fixtures.py

```diff
@@ -80,7 +80,7 @@ def generic_catalog_item(request, appliance, dialog_modscope, catalog_modscope):
 
 
 def create_catalog_item(appliance, provider, provisioning, dialog, catalog,
-        console_test=False):
+         vm_count='1', console_test=False):
```

Comment:
Why does this default to a string? I would expect it to be an `int`

Reply:
@john-dupuy , The Catalog item creation form expects the value assigned to VM count to be a string.